src/components/numberstable/NumbersTable.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import './NumbersTable.css'
+
+const NumbersTable = (props) => {
+
+    const numbers = [...Array(props.limit +1).keys()]
```

Comment:
:exploding_head: 

Damnnn son! What happens to the for loop and push number inside an empty array? 

Nice job professor :D